test/matchers/equals.spec.js

```diff
@@ -0,0 +1,38 @@
+var EqualMatcher = require('../../lib/matchers/equals');
+
+describe('equals matcher', function() {
+
+  it('fails when expectedValue is undefined', function() {
+    new EqualMatcher().match('', undefined).should.have.error(/should be defined/);
+  });
+
+  it('fails when expectedValue is an object', function() {
+    new EqualMatcher({}).match('', {}).should.have.error(/should be a primitive/);
```

Comment:
I was checking that they have given us a primitive to do the check, and that it IS a primitive. the test is that the expected value s.equals(<this_value>) has been set to a primitive.

Any primitive is OK